contracts/Conference.sol

```diff
@@ -57,14 +57,20 @@ contract Conference is Destructible, GroupAdmin {
      * @param _limitOfParticipants The number of participant. The default is set to 20. The number can be changed by the owner of the event.
      * @param _coolingPeriod The period participants should withdraw their deposit after the event ends. After the cooling period, the event owner can claim the remining deposits.
      * @param _encryption A pubic key. The admin can use this public key to encrypt pariticipant username which is stored in event. The admin can later decrypt the name using his/her private key.
+     * @param _owner Who the owner of this contract should be
      */
     constructor (
         string _name,
         uint256 _deposit,
         uint _limitOfParticipants,
         uint _coolingPeriod,
-        string _encryption
+        string _encryption,
+        address _owner
     ) public {
+        if (_owner != address(0)) {
+            owner = _owner;
+        }
```

Comment:
What do you think of getting rid of https://github.com/noblocknoparty/blockparty-contracts/blob/1a6f355b5134fc9133a447a0010e09753aca2b20/contracts/zeppelin/ownership/Ownable.sol#L21

and specify owner here explicitly like this.

```
        if (_owner != address(0)) {
            owner = _owner;
        }else{
            owner = msg.sender;
        }
```

If inheritance works like `Ownable#constructor` => `Party#constructor` then we are setting value to `owner` twice (and I am guessing this is how owner is falling back to `msg.owner` if 0 address is passed ). Doing this way, we only set it once and may potentially save gas cost as well as we don't have to worry about how inheritance order works.